smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/PackageManifestGenerator.kt

```diff
@@ -27,49 +30,64 @@ class PackageManifestGenerator(val ctx: ProtocolGenerator.GenerationContext) {
                     writer.write(".library(name: \$S, targets: [\$S])", ctx.settings.moduleName, ctx.settings.moduleName)
                 }
 
-                val externalDependencies = dependencies.filter { it.getProperty("url", String::class.java).get().isNotEmpty() }
-                val dependenciesByURL = externalDependencies.distinctBy { it.expectProperty("url", String::class.java) }
+                val externalDependencies = dependencies.filter {
+                    it.getProperty("url", String::class.java).getOrNull() != null ||
+                        it.getProperty("scope", String::class.java).getOrNull() != null
+                }
+                val dependenciesByURL = externalDependencies.distinctBy {
+                    it.getProperty("url", String::class.java).getOrNull()
+                        ?: "${it.getProperty("scope", String::class.java).get()}.${it.packageName}"
+                }
```

Comment:
"External dependencies" are written to the target dependencies section.
"Dependencies by URL" are written to the package dependencies section.